dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -105,18 +104,18 @@ public EntryResource(ToolDAO toolDAO, DockstoreWebserviceConfiguration configura
         topicsApi = new TopicsApi(apiClient);
     }
 
-    @PUT
+    @POST
     @Timed
     @UnitOfWork
     @Override
     @Path("/{id}/aliases")
     @ApiOperation(nickname = "updateAliases", value = "Update the aliases linked to a entry in Dockstore.", authorizations = {
```

Comment:
This is probably why the call above is updateAliases -- you need to change the nickname.

Reply:
fixed